mobify/platform-scaffold
# Loads app in parallel with the service worker

Loads app in parallel with the service worker

## Changes
- Currently, all of our files necessary for the rendering of the app don't get loaded until *after* the Service Worker downloads. This hurts TTI by [what seems to be upwards of 2.5 seconds](http://www.webpagetest.org/result/170626_ZN_1CE1/)!
- Also loads web push in parallel with the app so that it's compilation + execution doesn't cause blocking later.
- Also ensures that you don't need the SW to successfully install in order to use the app.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- @Cdok  ensure that your issue doesn't happen anymore.
